### PR TITLE
Mark/signup signin UI

### PR DESCRIPTION
**Pull request by mgkram4**
**What is the new behavior that this PR brings (describe or link to an issue)?**

UI for login/signup. Also added the tailwind config for the color palette. 

**PR checklist (requester)**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Your code is documented

**PR checklist (reviewer)**

// TODO

**Other information**:

### CODE REVIEW

File: .next/server/app-paths-manifest.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
```
Comment (blbergo) [inline .next/server/app-paths-manifest.json]: I think .next wasn't included in the git ignore

File: src/app/signin/page.tsx [hunk 2]
```diff
@@ -34,16 +49,26 @@ export default function SignIn({ searchParams }: SignUpProps) {
...
+            <hr className="border-gray-400" />
+          </div>
 
+          <GoogleSignInButton isSignUp />
+          <XSignInButton isSignUp />
           <p className="text-red-500 text-center font-bold">
             {searchParams?.error}
           </p>
         </form>
+        <Link href="/">
```
Comment (blbergo) [inline src/app/signin/page.tsx]: Should this point to ROUTES.SIGNUP?

**Review by yzerr0 — COMMENTED** [review]
looks pretty good! not sure what the `server-reference-manifest.json` and some other changes are so ill leave it to bryan to make the final approval.

File: src/app/signup/page.tsx [hunk 3]
```diff
@@ -36,15 +44,34 @@ export default async function SignUp({ searchParams }: SignUpProps) {
...
+            <hr className="border-gray-400" />
+          </div>
+
           <GoogleSignInButton isSignUp />
           <XSignInButton isSignUp />
           <p className="text-red-500 text-center font-bold">
             {searchParams?.error}
           </p>
         </form>
+        <Link href="/">
```
Comment (blbergo) [inline src/app/signup/page.tsx]: Same here, but with ROUTES.SIGNIN

File: tailwind.config.js [hunk 4]
```diff
@@ -7,6 +7,9 @@ module.exports = {
   theme: {
     extend: {
       colors: {
+        primary: "#266F35",
```
Comment (blbergo) [inline tailwind.config.js:10]: Nice job, later on we can add primary-500, primary-400 etc as we need more shades

File: src/app/signin/page.tsx [hunk 5]
```diff
@@ -1,6 +1,8 @@
 import { signInWithEmailAction } from "@/src/actions/auth/signup/actions";
 import XSignInButton from "@/src/components/button/x-signin";
```
Comment (blbergo) [inline src/app/signin/page.tsx]: I think this was from my previous PR but we can also use @components here instead of src/components

**Review by blbergo — APPROVED** [review]
Lgtm, left a few comments.

**Review by thatfoogabriel — APPROVED** [review]
Looks good to me, also confused on the .next files since its in the gitignore but the same files are tracking on my local repo too

File: .next/server/app-paths-manifest.json (hunk 1 above)
Comment (mgkram4) [reply .next/server/app-paths-manifest.json]: Will be honest not sure what  I did to affect this file, I was trying to do some error handling on the signup server component but needed state for the solution I was trying, so maybe in that process it affected it? I just checked the gitnore file and it has,  # next.js /.next/ /out/. First thoughts are to revert changes to this file.

File: src/app/signin/page.tsx (hunk 5 above)
Comment (mgkram4) [reply src/app/signin/page.tsx]: will fix

File: src/app/signup/page.tsx (hunk 3 above)
Comment (mgkram4) [reply src/app/signup/page.tsx]: yeah missed the other one two will fix

**Comment by mgkram4** [comment]
> Looks good to me, also confused on the .next files since its in the gitignore but the same files are tracking on my local repo too

Yeah also confused myslef, will be looking into it more, but thank you for letting me know about your repo

File: .next/server/app-paths-manifest.json (hunk 1 above)
Comment (blbergo) [reply .next/server/app-paths-manifest.json]: not ur fault most likely

**Review by blbergo — APPROVED** [review]
good to merge

**Pull request merged**